packages/grafana-prometheus/src/querybuilder/parsing.ts

```diff
@@ -251,16 +251,17 @@ function handleFunction(expr: string, node: SyntaxNode, context: Context) {
 
   const op = { id: funcName, params };
   // We unshift operations to keep the more natural order that we want to have in the visual query editor.
-  visQuery.operations.unshift(op);
```

Comment:
I think we should not change this. We do unshift to preserve the natural order. See the line above.

Reply:
instead of pushing I'd prefer unshifting because of the reason above